exonum/tests/config.rs

```diff
@@ -186,12 +205,24 @@ fn override_validators_count(config: &str, n: usize, folder: &str) {
         .expect("Create temp config file is failed");
 }
 
+fn copy_file_to_temp(file: &str, folder: &str) {
+    let source_file = full_testdata_name(file);
+    let destination_file = full_tmp_name(file, folder);
+    let contents = fs::read(source_file).unwrap();
+    let mut open_options = OpenOptions::new();
+    open_options.create(true).write(true);
+    #[cfg(unix)]
+    open_options.mode(0o600);
```

Comment:
Uh... why do we need to enforce the access mode on UNIX?

Reply:
@ilammy for windows it doesn't matter, does it ?

Reply:
Well, this line will not be compiled on Windows. `OpenOptions` on Windows does not even have a `mode()` method, just as Windows does not have a concept of the _access mode_ in UNIX sense.

I'm curious why do we need to do this on UNIX instead of simply using `File::create(path)`.

I kinda suspect that this is because we may validate the access mode for key files, so we need to have them to have an acceptable mode after being copied to /tmp. Though it seems ugly to do this in what looks like a general-purpose copying utility.

Reply:
@ilammy Because I couldn't change a file mode after it creation. Only this ugly approach let to create file with a custom file mode.

Reply:
Well, we cannot enforce custom file access mode on Windows, so why should we do this on Linux?

Reply:
@vitvakatu because we don't support windows os officially, and linux (and other os from unix family) we do.